A66-otel-stats.md

```diff
@@ -243,6 +243,14 @@ type MetricsOptions struct {
   // to Named Meter instances to instrument an application. To enable metrics
   // collection, set a meter provider. If unset, no metrics will be recorded.
   MeterProvider metric.MeterProvider
+  // TargetAttributeFilter is a callback that takes the target string and
+  // returns a bool representing whether to use target as a label value or use
+  // the string "other". If unset, will use the target string as is.
+  TargetAttributeFilter func(string) bool
+  // MethodAttributeFilter is a callback that takes the method string and
+  // returns a bool representing whether to use method as a label value or use
+  // the string "other". If unset, will use the method string as is.
+  MethodAttributeFilter func(string) bool
```

Comment:
You should note that this is only done for generic method types and not registered method types

Reply:
Done.